plugins/search/src/components/SearchType/SearchType.Tabs.tsx

```diff
@@ -20,7 +20,7 @@ import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import { makeStyles } from '@material-ui/core/styles';
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme: any) => ({
```

Comment:
Can we skip using `any` here and use `Theme` instead from `@material-ui/core/styles`

Reply:
Oh yeah for sure! Sorry I missed this.